### PR TITLE
chore: configure codesandbox to build storybook

### PR DESCRIPTION
**Pull request by jhoward-eightfold**
## CHANGE TYPE:

-   [ ] Bugfix Pull Request
-   [ ] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [ ] I have added unittests for this change

### CODE REVIEW

File: .codesandbox/ci.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
-    "sandboxes": ["new", "vanilla"],
```
Comment (dkilgore-eightfold) [inline .codesandbox/ci.json:2]: Please add this back to fix CodeSandbox CI.

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: .codesandbox/ci.json (hunk 1 above)
Comment (dkilgore-eightfold) [reply .codesandbox/ci.json:2]: Will also be needed once we figure out how to get CodeSandbox links in Storybook :)

**Review by dkilgore-eightfold — APPROVED** [review]

File: .codesandbox/ci.json [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+    "buildCommand": "build-ci",
+    "sandboxes": ["@eightfold.ai/octuple", "/storybook-static"],
```
Comment (dkilgore-eightfold) [inline .codesandbox/ci.json:3]: Looks like this config might need to be updated.

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: .codesandbox/ci.json (hunk 2 above)
Comment (jhoward-eightfold) [reply .codesandbox/ci.json:3]: Seems to be expecting a package.json in storybook-static. Unfortunately, documentation is on this config is rather light
https://codesandbox.io/docs/ci#configuration

**Pull request closed without merging**